Buổi 10/homework.js

```diff
@@ -0,0 +1,146 @@
+//Bài 1
+const Multi = number =>{
+    for(i=1;i<11;i++){

```

Comment:
1. Nhìn kỹ cách đặt dấu cách
2. Khai báo biến bắt buộc phải dùng từ khóa khai báo, trong trường hợp này `i` bị thay đổi do vậy đặt là `let`
```suggestion
    for (let i=1; i < 11; i++) {
```